src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -923,6 +932,10 @@
 examples:
   - name: Validate whether a template is valid at tenant scope.
     text: az deployment tenant validate --location WestUS --template-file {template-file}
+  - name: Validate whether a template is valid at tenant scope. (autogenerated)
+    text: |
+        az deployment tenant validate --location WestUS --name mydeployment --parameters MyValue=This MyArray=@array.json --template-file azuredeploy.json
```

Comment:
Same as above

Reply:
Could you apply suggestions as well?

Reply:
How about @myparameters.json ?
It's consistent with az deployment create --location WestUS --template-uri https://myresource/azuredeploy.json --parameters @myparameters.json

Reply:
```suggestion
        az deployment tenant validate --location WestUS --name mydeployment --parameters @myparameters.json --template-file azuredeploy.json
```